FirebaseAppDistribution/Sources/FIRAppDistribution.m

```diff
@@ -240,28 +262,27 @@ - (UIWindow *)createUIWindowForLogin {
 - (void)handleReleasesAPIResponseWithData:data
                                completion:(FIRAppDistributionUpdateCheckCompletion)completion {
   NSError *error = nil;
-  NSDictionary *object = [NSJSONSerialization
-                          JSONObjectWithData:data
-                          options:0
-                          error:&error];
-
-    NSArray *releaseList = [object objectForKey:@"releases"];
-    for (NSDictionary *releaseDict in releaseList) {
-        if([[releaseDict objectForKey:@"latest"] boolValue]) {
-            NSString *codeHash = [releaseDict objectForKey:@"codeHash"];
-            NSString *executablePath = [[NSBundle mainBundle] executablePath];
-            FIRAppDistributionMachO *machO = [[FIRAppDistributionMachO alloc] initWithPath:executablePath];
-
-            if(![codeHash isEqualToString:machO.codeHash]) {
-                FIRAppDistributionRelease *release = [[FIRAppDistributionRelease alloc] initWithDictionary:releaseDict];
-                dispatch_async(dispatch_get_main_queue(), ^{
-                    completion(release, nil);
-                });
-            }
+  NSDictionary *object = [NSJSONSerialization JSONObjectWithData:data options:0 error:&error];
+
+  NSArray *releaseList = [object objectForKey:@"releases"];
+  for (NSDictionary *releaseDict in releaseList) {
+    if ([[releaseDict objectForKey:@"latest"] boolValue]) {
+      NSString *codeHash = [releaseDict objectForKey:@"codeHash"];
+      NSString *executablePath = [[NSBundle mainBundle] executablePath];
+      FIRAppDistributionMachO *machO =
+          [[FIRAppDistributionMachO alloc] initWithPath:executablePath];
+
+      if (![codeHash isEqualToString:machO.codeHash]) {
+        FIRAppDistributionRelease *release =
+            [[FIRAppDistributionRelease alloc] initWithDictionary:releaseDict];
+        dispatch_async(dispatch_get_main_queue(), ^{
+          completion(release, nil);
```

Comment:
Is it ok that this sometimes is never called?

Reply:
This method should be pretty easy to write tests for

Reply:
Will follow up in a subsequent PR to this feature branch